aws/cloudformation-templates/services/service/_template.yaml

```diff
@@ -178,6 +195,10 @@ Parameters:
     Type: String
     Description: Lambda Arn for deleting ECR repository
 
+  UseDefaultIvsStreams:
```

Comment:
NIT: Can we use a standard convention across with either uppercase IVS everywhere or go with Ivs?

Reply:
We will update the CloudFormation to use uppercase IVS.